monitoring/mongodb/dashboard.py

```diff
@@ -143,177 +144,177 @@ class Metrics:
         "job",
         "rs_state",
         namespace="${namespace}",
-    ).with_defaults(f'job=~"{JOB_PATTERN}"')
+    ).with_defaults(DATABASE_FILTER_USER, JOB_FILTER, rs_state=RS_STATE_PRIMARY)
```

Comment:
nit: FYI, `with_defaults` is kind of the initial solution to provide default value, but it requires duplicating the name of the field. The preferred approach -when possible- is to use named parameters to provide both the field name and value, like what is done for `namespace`:

```suggestion
        rs_state=RS_STATE_PRIMARY,
        namespace="${namespace}",
    ).with_defaults(DATABASE_FILTER_USER, JOB_FILTER)
```